builders/tools/explorers.md

```diff
@@ -61,22 +61,34 @@ As main features, Blockscout offers:
 
 ![Blockscout Explorer](/images/builders/tools/explorers/explorers-2.png)
 
+### Expedition {: #expedition } 
+
+A Moonbeam-themed version of the [Expedition](https://github.com/xops/expedition){target=_blank} explorer can be found in [this link](https://moonbeam-explorer.netlify.app/){target=_blank}.
+
+By default, the explorer is connected to the Moonriver. However, you can switch to another network or connect it to a local dev node by following the next steps:
+
+ 1. Click on the gear icon in the top right corner
+ 2. Select "Development" if you have a node running on `{{ networks.development.rpc_url }}` (default RPC location of a Moonbeam node running with `--dev`flag). You can also switch back to "Moonbase Alpha"
```

Comment:
doing a drive-by haha sorry... I've been slowly trying to convert to using bold fonts for UI items across the docs site... so like instead of "Development" it would be **Development**

Reply:
taking some things from the google documentation style guide 🙂  https://developers.google.com/style/text-formatting